lib/web_ui/lib/src/engine/shadow.dart

```diff
@@ -45,12 +45,12 @@ const double kLightOffsetY = -400.0;
 /// This is not a complete physical model. For example, this does not take into
 /// account the size of the shape (this function doesn't even take the shape as
 /// a parameter). It's just a good enough approximation.
-ui.Offset computeShadowOffset(double elevation) {
+ui.Offset computeShadowOffset(double? elevation) {
```

Comment:
Called from 2 places that look non-null. non-nullable ?

Reply:
Done.